lib/rules/web/admin_console/4.9/logging_kibana.xyaml

```diff
@@ -195,3 +196,37 @@ click_visuallize_button_in_discover_page:
     timeout: 30
     op: click
 
+kibana_index_pattern_exist:
+  action: go_to_kibana_management_page
+  elements:
+  - selector:
+      xpath: //span[@class="euiSideNavItemButton__label" and contains(., "Index Patterns")]
+    op: click
+    timeout: 30
+  - selector:
+      xpath: //div[@id="indexPatternListReact"]//*[text()="<index_pattern_name>"]
+
+select_index_pattern_in_kibana:
+  action: go_to_kibana_discover_page
+  action:
+    if_element:
+      selector:
+        xpath: //span[@class="euiSideNavItemButton__label" and contains(., "Index Patterns")]
```

Comment:
@anpingli I can't find this element in `Discover` page, from line 203, I think it should be in the `Management` page.